crypto/scrypt.go

```diff
@@ -0,0 +1,174 @@
+package crypto
+
+// Params describes the input parameters to the scrypt
+import (
+	"bytes"
+	"crypto/subtle"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"golang.org/x/crypto/scrypt"
+)
+
+// The code below is heavily inspired by https://github.com/elithrar/simple-scrypt
+
+// Scrypt params
+// TODO audit this parameters taken from simple-scrypt DefaultParams
+const n = 16384
+const r = 8
+const p = 1
```

Comment:
nitpick: we could rename those const to `defaultN`, `defaultR`, (or anything a bit more verbose...) to make sure they are not "shadowed" in a function and see more easily where they are actually used.